src/Stack/Build/Installed.hs

```diff
@@ -39,17 +44,33 @@ data GetInstalledOpts = GetInstalledOpts
       -- ^ Require debugging symbols?
     }
 
+toInstallMap :: MonadIO m => SourceMap -> m InstallMap
+toInstallMap sourceMap = do
+    let loadVersion loc common = do
```

Comment:
Maybe renaming to `loadLocalVersion` and removing `loc` argument? Would be slightly clearer, considering it's only ever called with `Local`

Reply:
The point is that it's not exclusively about locals, looks like the code got copy-pasted and `versionMaybeFromPM` shouldn't use the same code but should just use loadVersion - I'll refactor it